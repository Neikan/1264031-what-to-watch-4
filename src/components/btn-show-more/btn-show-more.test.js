import React from "react";
import renderer from "react-test-renderer";
import BtnShowMore from "./btn-show-more";


describe(`Test BtnShowMore component`, () => {
  test(`BtnShowMore component is created and rendered correctly`, () => {
    const tree = renderer.create(
        <BtnShowMore
          onSelect={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
