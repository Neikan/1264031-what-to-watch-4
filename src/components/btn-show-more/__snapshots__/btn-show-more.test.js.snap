// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test BtnShowMore component BtnShowMore component is created and rendered correctly 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
